pkg/render/common/securitycontextconstraints/security_context_constraints.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package securitycontextconstraints
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	ocsv1 "github.com/openshift/api/security/v1"
+
+	"github.com/tigera/operator/pkg/ptr"
+)
+
+const (
```

Comment:
If I understand it correctly, these are all predefined constraints that openshift already understands by default, is that right? Perhaps it is worth adding a comment to that effect, it would have helped me understand it more quickly.

Reply:
Added a doc reference to the default OpenShift SCCs.